spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -663,13 +663,23 @@ dataflow:>task execution list
 dataflow:>task destroy --name task1
 ```
 
-== Application Configuration
+== Application and Server Properties
 
-This section covers how you can customize the deployment of your applications. You can use a number of deployer properties to influence settings for the applications that are deployed.
+This section covers how you can customize the deployment of your applications. You can use a number of properties to influence settings for the applications that are deployed. Properties can be applied on a per application basis or in the server configuration for all deployed applications.
+
+NOTE: Properties set on a per application basis will always take precedence over properties set the server configuration. This allows for the ability to override property values set at the server level.
```

Comment:
"set the server configuration" -> "set as the server configuration"

Reply:
"This allows for the ability to override property values set at the server level" -> "This allows for the ability to override global server level properties on a per-application basis."